docs/splatalogue/splatalogue.rst

```diff
@@ -1,5 +1,3 @@
-.. doctest-skip-all
```

Comment:
Interesting why the examples work in this without remote-data? Would it make sense to make it opt-out to use the shipped json?

Reply:
Many of them are labeled remote-data; only a couple work without it.